test/Claimer.t.sol

```diff
@@ -27,96 +36,111 @@ contract ClaimerTest is Test {
-        winners[0] = winner1;
-        uint8[] memory tiers = new uint8[](1);
-        tiers[0] = 0;
-        // fee should be the target rn
-        mockPrizePool(100, 1000, 0, 0);
-        vm.expectRevert("insuff fee");
-        claimer.claimPrizes(vault, winners, tiers, 1e18, address(this));
+    function testClaimPrizes_single() public {
+        Claim[] memory claims = new Claim[](1);
+        claims[0] = Claim({
+            vault: vault,
+            winner: winner1,
+            tier: 1
+        });
+        mockPrizePool(1, -100, 0);
+        mockClaimPrize(claims[0].winner, 1, claims[0].winner, uint96(UNSOLD_100_SECONDS_IN_FEE), address(this), 100);
+        (uint256 claimCount, uint256 totalFees) = claimer.claimPrizes(1, claims, address(this));
+        assertEq(claimCount, 1, "Number of prizes claimed");
+        assertEq(totalFees, UNSOLD_100_SECONDS_IN_FEE, "Total fees");
     }
 
-    function testClaimPrizes_fees() public {
-        address[] memory winners = new address[](2);
-        winners[0] = winner1;
-        winners[1] = winner2;
-        uint8[] memory tiers = new uint8[](2);
-        tiers[0] = 1;
-        tiers[1] = 1;
-        mockPrizePool(1000, 1000, -1, 0);
-        uint256 totalFees = TARGET_PRICE + AHEAD1_PRICE;
-        mockClaimPrize(winners[0], 1, winners[0], uint96(totalFees/2), address(this), 100);
-        mockClaimPrize(winners[1], 1, winners[1], uint96(totalFees/2), address(this), 100);
-        uint256 fees = claimer.claimPrizes(vault, winners, tiers, 0.0001e18, address(this));
-        assertEq(fees, totalFees);
+    function testClaimPrizes_multiple() public {
+        Claim[] memory claims = new Claim[](2);
+        claims[0] = Claim({
+            vault: vault,
+            winner: winner1,
+            tier: 1
+        });
+        claims[1] = Claim({
+            vault: vault,
+            winner: winner2,
+            tier: 1
+        });
+        mockPrizePool(1, -100, 0);
+        mockClaimPrize(claims[0].winner, 1, claims[0].winner, uint96(UNSOLD_100_SECONDS_IN_FEE), address(this), 100);
+        mockClaimPrize(claims[1].winner, 1, claims[1].winner, uint96(SOLD_ONE_100_SECONDS_IN_FEE), address(this), 100);
+        (uint256 claimCount, uint256 totalFees) = claimer.claimPrizes(1, claims, address(this));
+        assertEq(claimCount, 2, "Number of prizes claimed");
+        assertEq(totalFees, UNSOLD_100_SECONDS_IN_FEE + SOLD_ONE_100_SECONDS_IN_FEE, "Total fees");
     }
 
-    function testEstimateFees_zero() public {
-        mockPrizePool(1000, 1000, 0, 0);
-        assertEq(claimer.estimateFees(0), 0);
+    function testClaimPrizes_maxFee() public {
+        Claim[] memory claims = new Claim[](1);
+        claims[0] = Claim({
+            vault: vault,
+            winner: winner1,
+            tier: 1
+        });
+        mockPrizePool(1, -1, 0);
+        mockLastCompletedDrawStartedAt(-80000); // much time has passed, meaning the fee is large
```

Comment:
Wouldn't the max fee be hit at `-1 * (1 days - 1 seconds)`?

Reply:
It's not trying to warp to the exact max time; just at a time when the max fee is guaranteed to be selected